komfydb/common/tuple.cc

```diff
@@ -8,18 +8,31 @@
 namespace {
 
 using komfydb::common::Field;
+using komfydb::common::IntField;
+using komfydb::common::StringField;
 
-template <typename T>
-bool compare_fields(const Field* f1, const Field* f2) {
-  T v1, v2;
+bool compare_fields(const IntField* f1, const IntField* f2) {
   if ((f1 == nullptr && f2 != nullptr) || (f1 != nullptr && f2 == nullptr)) {
```

Comment:
This checks can be done easier:`if (f1 == f2) return true; if (f1 == nullptr || f2 == nullptr) return false;`